client/libra_wallet/src/error.rs

```diff
@@ -8,7 +8,7 @@ use std::{convert, error::Error, fmt, io};
 /// We define our own Result type in order to not have to import the libra/common/failure_ext
 pub type Result<T> = ::std::result::Result<T, WalletError>;
 
-/// Libra Wallet Error is a convenience enum for generating arbitrary WalletErrors. Currently, only
+/// Libra Wallet Error is a convenience enum for generating arbitary WalletErrors. Currently, only
```

Comment:
This doesn't look right.

Reply:
@wqfish thanks! yes you are correct. changed it to the correct spelling